tests/extras/datasets/pandas/test_sql_dataset.py

```diff
@@ -11,6 +12,7 @@
 TABLE_NAME = "table_a"
 CONNECTION = "sqlite:///kedro.db"
 SQL_QUERY = "SELECT * FROM table_a"
+EXECUTION_OPTIONS = {"streaming": True}
```

Comment:
Should this be `stream_results` as well?

Reply:
Yes, it should be